### ISSUE TITLE
.jonsl Error: Extra Data, Line 2 Col 1 in BatchTransform

### ISSUE THREAD

**Issue by cheeseki** [huggingface/notebooks#187]
Followed the notebook https://github.com/huggingface/notebooks/blob/main/sagemaker/12_batch_transform_inference/sagemaker-notebook.ipynb for input files preprocess, but the generated 'tweet_data.jsonl' cannot be read by Sagemaker, the error message is related to json decoder, "Extra Data, Line 2 Col 1". Does anyone else experience same issue?

**Comment by krpapadopoulos** [comment]
Yes, i am using the template in the custom inference for a sentence transformer model.  The input file is prepared as jsonl and the batch transform is used as indicated in the blog post:  https://huggingface.co/docs/sagemaker/inference for batch transform.

I did tests and see that the output file (jsonl.output) is not organized into jsonl that is why there is a JSON decoding error likely when trying to read it.  There are no line breaks between the json outputs.
